tests/acceptance/FormEditorTextInputStylesCest.php

```diff
@@ -35,12 +35,24 @@ public function changeTextInputStyles(\AcceptanceTester $i) {
     // Check element has styles
     $i->assertAttributeContains('[data-automation-id="editor_first_name_input"]', 'style', 'border-width: 10px;');
     $i->assertAttributeContains('[data-automation-id="editor_first_name_label"]', 'style', 'font-weight: bold;');
+
     // Apply to all
     $i->click('[data-automation-id="styles_apply_to_all"]');
-    // Check email block has styles too
+
+    // Add heading block and write some title
+    $i->addFromBlockInEditor('Heading');
+    $i->fillField('[data-title="Heading"]', 'Lorem Ipsum');
+    $i->see('Lorem Ipsum');
+
+    // Add paragraph block and write some text
+    $i->addFromBlockInEditor('Paragraph');
+    $i->fillField('[data-title="Paragraph"]', 'Lorem ipsum dolor sit amet');
+    $i->see('Lorem ipsum dolor sit amet');
```

Comment:
Should we also check that those new elements are present in the editor after the reload below? And also check that these elements are present in the rendered page?